### ISSUE TITLE
Error "y1 must be greater than or equal to y0"

### ISSUE THREAD

**Issue by jellewie** [Skons/hassio-addons#40]
When I try and set up this plugin it does not seem to work getting the data.
`Unable to process station_request payload '{'fuel_type': 'euro95', 'radius': 5, 'latitude': 53, 'longitude': 6, 'to_publish': 3}' with error: 'y1 must be greater than or equal to y0'`

I have tried look in the readme's and the documentation, but no luck. Also it seems the codebase does not include those words anywhere so sadly I can't read it back and reverse figure out what is wrong. I hope you can help me.

debug Log:
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Add-on: Dutch gas prices
 Get gas prices from dutch gas stations
-----------------------------------------------------------
 Add-on version: 2022.3.25.1
 You are running the latest version of this add-on.
 System: Home Assistant OS 9.5  (aarch64 / raspberrypi4-64)
 Home Assistant Core: 2023.4.2
 Home Assistant Supervisor: 2023.04.0
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
Log level is set to DEBUG
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[13:14:26] DEBUG: Requested API resource: http://supervisor/services/mqtt
[13:14:26] DEBUG: Request method: GET
[13:14:26] DEBUG: Request data: {}
[13:14:26] DEBUG: API HTTP Response code: 200
[13:14:26] DEBUG: API Response: {"result": "ok", "data": {"host": "core-mosquitto", "port": 1883, "ssl": false, "protocol": "3.1.1", "username": "addons", "password": "mea2eCoomeid8vu7shetae4aemohghiebohwahVoosiesohghix6xukaeji1cieM", "addon": "core_mosquitto"}}
[13:14:27] INFO: Log_level: debug
[13:14:27] INFO: mqtt_host: core-mosquitto
[13:14:27] INFO: mqtt_port: 1883
[13:14:27] DEBUG: mqtt_username: mqtt-user
[13:14:27] DEBUG: mqtt_password: 1234
[13:14:27] INFO: Starting dutch gas prices...
[2023-04-13 13:14:28] INFO    : DGP initialized, launching client
[2023-04-13 13:14:28] INFO    : Connecting mqtt
[2023-04-13 13:14:28] INFO    : Connected with result code 0
[2023-04-13 13:17:18] INFO    : Received payload '{"fuel_type":"euro95","radius":5,"latitude":53,"longitude":6, "to_publish":3}' on topic 'dgp/gas_stations'
[2023-04-13 13:17:18] DEBUG   : gas_station: Fuel 'euro95' new request
[2023-04-13 13:17:18] DEBUG   : Starting new HTTPS connection (1): tankservice.app-it-up.com:443
[2023-04-13 13:17:18] DEBUG   : https://tankservice.app-it-up.com:443 "GET /Tankservice/v1/places?fmt=web&fuel=euro95 HTTP/1.1" 200 None
[2023-04-13 13:17:18] DEBUG   : station: Station id '6899' new request
[2023-04-13 13:17:18] DEBUG   : Starting new HTTPS connection (1): tankservice.app-it-up.com:443
[2023-04-13 13:17:19] DEBUG   : https://tankservice.app-it-up.com:443 "GET /Tankservice/v1/places/6899.png HTTP/1.1" 200 None
[2023-04-13 13:17:20] ERROR   : Unable to process station_request payload '{'fuel_type': 'euro95', 'radius': 5, 'latitude': 53, 'longitude': 6, 'to_publish': 3}' with error: 'y1 must be greater than or equal to y0'
[2023-04-13 13:17:20] INFO    : publishing status to mqtt
[2023-04-13 13:17:20] DEBUG   : on_message: received the topic 'homeassistant/sensor/dgp/status/config'
[2023-04-13 13:17:21] DEBUG   : publish_status: Waiting for discovery on topic 'homeassistant/sensor/dgp/status/config'
[2023-04-13 13:17:21] DEBUG   : publish_status: Autodiscover 'True', State 'True' and Attributes 'True'
[2023-04-13 13:17:21] INFO    : publishing status done
```

It does that it can get the Json string of stations `https://tankservice.app-it-up.com/Tankservice/v1/places?fmt=web&fuel=euro95`
and the image page for the station if found also works `https://tankservice.app-it-up.com/Tankservice/v1/places/6899.png`
But then the bug starts I think

**Comment by ZeroZXD** [comment]
Hello, Just adding to this. 

Just installed the add-on yesterday and i'm also experiencing the same error in the debug logs.
I attached my debug log below, its the same error as above, but running on amd64 hardware instead of aarch64 (don't think it matters).

[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Add-on: Dutch gas prices
 Get gas prices from dutch gas stations
-----------------------------------------------------------
 Add-on version: 2022.3.25.1
 You are running the latest version of this add-on.
 System: Home Assistant OS 9.5  (amd64 / qemux86-64)
 Home Assistant Core: 2023.4.4
 Home Assistant Supervisor: 2023.04.0
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
Log level is set to DEBUG
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[08:47:09] DEBUG: Requested API resource: http://supervisor/services/mqtt
[08:47:09] DEBUG: Request method: GET
[08:47:09] DEBUG: Request data: {}
[08:47:09] DEBUG: API HTTP Response code: 200
[08:47:09] DEBUG: API Response: {"result": "ok", "data": {"host": "core-mosquitto", "port": 1883, "ssl": false, "protocol": "3.1.1", "username": "addons", "password": "baht0Gief3hiTe6boaMeigh6ahfaht5queebaiXie8beefuiD3finahw3chah2ce", "addon": "core_mosquitto"}}
[08:47:09] INFO: Log_level: debug
[08:47:09] INFO: mqtt_host: core-mosquitto
[08:47:09] INFO: mqtt_port: 1883
[08:47:09] DEBUG: mqtt_username: mqtt-user
[08:47:09] DEBUG: mqtt_password: HIDDEN
[08:47:09] INFO: Starting dutch gas prices...
[2023-04-15 08:47:09] INFO    : DGP initialized, launching client
[2023-04-15 08:47:09] INFO    : Connecting mqtt
[2023-04-15 08:47:09] INFO    : Connected with result code 0
[2023-04-15 08:47:25] INFO    : Received payload '{"fuel_type":"euro95","radius":5,"latitude":HIDDEN,"longitude":HIDDEN,"to_publish":3}' on topic 'dgp/gas_stations'
[2023-04-15 08:47:25] DEBUG   : gas_station: Fuel 'euro95' new request
[2023-04-15 08:47:25] DEBUG   : Starting new HTTPS connection (1): tankservice.app-it-up.com:443
[2023-04-15 08:47:25] DEBUG   : https://tankservice.app-it-up.com:443 "GET /Tankservice/v1/places?fmt=web&fuel=euro95 HTTP/1.1" 200 None
[2023-04-15 08:47:26] DEBUG   : station: Station id '4668' new request
[2023-04-15 08:47:26] DEBUG   : Starting new HTTPS connection (1): tankservice.app-it-up.com:443
[2023-04-15 08:47:26] DEBUG   : https://tankservice.app-it-up.com:443 "GET /Tankservice/v1/places/4668.png HTTP/1.1" 200 None
[2023-04-15 08:47:26] ERROR   : Unable to process station_request payload '{'fuel_type': 'euro95', 'radius': 5, 'latitude': HIDDEN, 'longitude': HIDDEN, 'to_publish': 3}' with error: 'y1 must be greater than or equal to y0'
[2023-04-15 08:47:26] INFO    : publishing status to mqtt
[2023-04-15 08:47:26] DEBUG   : on_message: received the topic 'homeassistant/sensor/dgp/status/config'
[2023-04-15 08:47:27] DEBUG   : publish_status: Waiting for discovery on topic 'homeassistant/sensor/dgp/status/config'
[2023-04-15 08:47:27] DEBUG   : publish_status: Autodiscover 'True', State 'True' and Attributes 'True'
[2023-04-15 08:47:27] INFO    : publishing status done

If you need more logs or information, let me know.

**Comment by Skons** [comment]
I can reproduce this with a fresh install. Probably an update of an dependency, will update as soon as possible.

**Comment by Skons** [comment]
Just released 2023.4.16.1 where this issue is solved
